examples/todo-list/src/__tests__/acceptance/todo-list.acceptance.ts

```diff
@@ -204,6 +204,17 @@ describe('TodoListApplication', () => {
     });
   });
 
+  it('exploded filter conditions work', async () => {
+    const list = await givenTodoListInstance(todoListRepo);
+    await givenTodoInstance(todoRepo, {title: 'todo1', todoListId: list.id});
+    await givenTodoInstance(todoRepo, {title: 'todo2', todoListId: list.id});
+    await givenTodoInstance(todoRepo, {title: 'todo3', todoListId: list.id});
+
+    const response = await client.get('/todos').query('filter[limit]=2');
```

Comment:
This is a bit redundant, because `query({filter: {limit: 2}})` is automatically converted by supertest to `?filter[limit]=2`. Having said that, I think it's a good idea to have tests that are explicitly verifying different encoding styles, as opposed to the current status, where such encodings are tested implicitly, see:

https://github.com/strongloop/loopback-next/blob/303d3ff6ebe751b238ab16e021f6a25f09f4a1be/examples/todo-list/src/__tests__/acceptance/todo-list.acceptance.ts#L110

https://github.com/strongloop/loopback-next/blob/303d3ff6ebe751b238ab16e021f6a25f09f4a1be/examples/todo-list/src/__tests__/acceptance/todo-list.acceptance.ts#L196-L198

Writing such tests is out of scope of this spike, but please do include a task/todo-item in the acceptance criteria of the story that will track the real implementation.